src/__tests__/__snapshots__/documenter.test.ts.snap

```diff
@@ -2156,6 +2156,25 @@ Do not use \`ariaLabel\` and \`ariaLabelledby\` at the same time.",
       "optional": true,
       "type": "CartesianChartProps.DetailPopoverFooter<T>",
     },
+    Object {
+      "description": "Specifies customization of the content displayed in the chart popover.
+Use this for wrapping keys or values in links, or to display an additional
+level of nested items.",
+      "inlineType": Object {
+        "name": "MixedLineBarChartProps.DetailPopoverSeriesContent",
+        "parameters": Array [
+          Object {
+            "name": "__namedParameters",
```

Comment:
Is this really expected to print it on our API documentation like this?

Reply:
Thanks for spotting this. Refactored the interface to fix this [here](https://github.com/cloudscape-design/components/pull/1644/commits/8216ba7eddbb632e53666f6d770a2387a39e3fb3).

Note: unfortunately we don't currently have a way to show all the named parameters in the API documentation, instead we show the single parent parameter. A similar example of this is the `renderAriaLive` function of the Table component, see https://cloudscape.design/components/table/?tabId=api#properties